app/src/modules/gql/apolloClient.js

```diff
@@ -0,0 +1,26 @@
+import { ApolloClient, ApolloLink, InMemoryCache, HttpLink } from 'apollo-boost'
+import VueApollo from 'vue-apollo'
+
+export default (uri) => {
+  const httpLink = new HttpLink({ uri })
+  const authLink = new ApolloLink((operation, forward) => {
+    // add the authorization to the headers
+    const token = localStorage.getItem('token') || null
+    operation.setContext({
+      headers: {
+        authorization: token ? `${token}` : ''
+      }
+    })
+    return forward(operation)
+  })
+
+  const apolloClient = new ApolloClient({
+    uri,
+    link: authLink.concat(httpLink), // Chain auth token with the HttpLink
+    cache: new InMemoryCache()
```

Comment:
I forgot why we added this, can we get rid of it?

Reply:
See my note in the PR: 
When I leave out link and cache, I get the following error: 
```
In order to initialize Apollo Client, you must specify 'link' and 'cache' properties in the options object.
These options are part of the upgrade requirements when migrating from Apollo Client 1.x to Apollo Client 2.x.
For more information, please visit: https://www.apollographql.com/docs/tutorial/client.html#apollo-client-setup
```

Reply:
Opened new issue #277